File: Programmers/Lv.1/옹알이 (2)/133499.java
class Solution {
    public int solution(String[] babbling) {
        int answer = 0;
        
        String[] arr = {"aya", "ye", "woo", "ma"};
        for (String b : babbling) {
            for (String a : arr) {
                if (b.contains(a.repeat(2))) break;
                b = b.replaceAll(a, " ");
            }
            
            if ("".equals(b.trim())) answer++;
        }
        
        return answer;
    }
}

File: Programmers/Lv.1/옹알이 (2)/133499.js
function solution(babbling) {
    let answer = 0;
    
    const arr = ["aya", "ye", "woo", "ma"];
    babbling.forEach(b => {
        arr.forEach(a => {
            if (b.includes(a.repeat(2))) return;
            b = b.replaceAll(a, " ");
        });
        
        if (b.trim() === "") answer++;
    });
    
    return answer;
}
